timeline.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Timeline</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Check consistency with indentation in the head tag :)

Reply:
This was done purposefully to separate out the open source stylesheets, added a comment for clarity